knownprojects_build/sql/review_project.sql

```diff
@@ -3,6 +3,7 @@ SELECT
     a.source,
     a.record_id,
     a.record_name,
+    a.dcb_projectdescription,
```

Comment:
why not just do a one line
`(select dcb_projectdescription from dcp_application where record_id = a.record_id) as dcb_projectdescription`
there's no need to change the combined table